dashboard/app.R

```diff
@@ -375,11 +400,33 @@ server <- function(input, output, session) {
       }
     }
 
+    # Title plot
+    if (input$scoreType == "wis") {
+      plotTitle = "Weighted Interval Score"
+    }
+    else if (input$scoreType == "sharpness") {
+      plotTitle = "Spread"
+    }
+    else if (input$scoreType == "ae") {
+      plotTitle = "Absolute Error"
+    }
+    else {
+      plotTitle = "Coverage"
+    }
+
     titleText = paste0('<b>', plotTitle,'</b>','<br>', '<sup>',
                        'Target Variable: ', input$targetVariable,
                        locationSubtitleText, '<br>',
                        tags$span(id="drag-to-zoom", " Drag to zoom"),
                        '</sup>')
+
+    # Some forecasters have multiple forecasts for the same week end date with the same score,
+    # and tsibbles needs unique keys/indexes
+    # So we remove dups before setting tsibble
+    # TODO this "fix" messes up the faceted plots, need to figure this out
```

Comment:
is this a blocking TODO? if not dump it in an issue and cite the link here

Reply:
Done.